commerce-cloud-code/app_yourshophere/cartridge/partials/tile/tile.js

```diff
@@ -13,7 +13,7 @@ exports.createModel = function createModel(input) {
  * @returns {string} The HTML template for the tile
  */
 exports.template = (model) => `
-<article data-include-url="${request.httpQueryString}" data-analytics='${model.analytics}' data-analytics-contribution='${model.analyticsContribution}'>
+<article class="product-tile" data-include-url="${request.httpQueryString}" data-analytics='${model.analytics}' data-analytics-contribution='${model.analyticsContribution}'>
```

Comment:
we don't have a product-grid class to target the article tag?
Do we need targetting at all? I.e. isn't article always a tile?